csharp-ovh/Client/Client.cs

```diff
@@ -51,31 +50,15 @@ namespace Ovh.Api
     /// <summary>
     /// Low level OVH Client. It abstracts all the authentication and request
     /// signing logic along with some nice tools helping with key generation.
-    /// All low level request logic including signing and error handling takes place
-    /// in "Client.Call" function. Convenient wrappers
-    /// "Client.Get" "Client.Post", "Client.Put",
-    /// "Client.Delete" should be used instead. "Client.Post",
-    /// "Client.Put" both accept arbitrary list of keyword arguments
-    /// mapped to "data" param of "Client.Call".
-    /// Example usage:
-    ///     from ovh import Client, APIError
-    ///     REGION = 'ovh-eu'
-    ///     APP_KEY= "&lt;application key&gt;"
-    ///     APP_SECRET= "&lt;application secret key&gt;"
-    ///     CONSUMER_KEY= "&lt;consumer key&gt;>"
-    ///     client = Client(REGION, APP_KEY, APP_SECRET, CONSUMER_KEY)
-    ///     try:
-    ///         print client.get('/me')
-    ///     except APIError as e:
-    ///         print "Ooops, failed to get my info:", e.msg
     /// </summary>
     public partial class Client
     {
         private readonly Dictionary<string, string> _endpoints =
             new Dictionary<string, string>();
```

Comment:
Maybe move to `Dictionary<string, Uri>`?

Reply:
I could, but why would be the benefit?